microhapdb/tests/test_cli.py

```diff
@@ -336,6 +337,7 @@ def test_ae_pop(capsys):
 mh18PK-87558 MHDBM-1e5374f1    GRCh38 chr18 1960542,1960557,1960561,1960566,1960582,1960588 1.9767 0.1325 0.0494 10.1016/j.fsigen.2018.05.008
 mh18USC-18pA MHDBM-56dfa93b    GRCh38 chr18         5280017,5280020,5280070,5280071,5280093 1.8402 0.2130 0.1818 10.1016/j.fsigen.2019.102213
   mh18CP-005 MHDBM-a85754d3    GRCh38 chr18                 8892864,8892893,8892896,8892907 3.3873 0.0904 0.0059                       ALFRED
+ mh18ZHA-004 MHDBM-db649dba    GRCh38 chr18             14315931,14315952,14315958,14316046 3.1131 0.1949 0.0118 10.1016/j.fsigen.2020.102255
```

Comment:
`mh18ZHA-004` is a new marker that has been added that is contained within the region of this test. It has been updated to include the new marker.